addons/message_subscription_expiring/isExpiring.js

```diff
@@ -0,0 +1,11 @@
+(function(api, condition) {
+  //Show message only if within 1 week of expiring.
+  let weekBeforeExpireSecs = (api.subscriptionData.expiresOn / 1000) - 7*24*60*60;
```

Comment:
technically `expiresOn` can be 0. So, `weekBeforeExpireSecs` will be -604800, which is less than `Date.now()`.

Reply:
The other side of that && expression should handle that case :+1: